cloudhub-2/modules/ROOT/pages/ps-config-app-level-egress.adoc

```diff
@@ -0,0 +1,171 @@
+= Configuring Application Level Egress Rules
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Application level egress rules are independent of private space level firewall rules. For an egress traffic to go through, it must pass both private space and application level control.
+
+
+== Before You Begin
+
+* Set up your private space and complete the provisioning of your private network.
+* Verify that your account has CloudHub Network Viewer permission, which is required to view a private space's firewall and application level rules.
+* Verify that your account has CloudHub Network Administrator permission, which is required to modify a private space's firewall and application level rules.
+
+== Configure Application Level Rules in Your Private Space
+
+To configure the application level groups in a private space:
+
+. From Anypoint Platform, select *Runtime Manager* > *Private Spaces*.
+. Click the name of the private space to manage.
+. On the *Firewall rules* tab, expand the *App-Level Rules* section.
+
+By default, the application level egress rules feature is disabled when you create a new private space. Toggle the feature to enable it.
+
+[NOTE]
+Ensure that you have completed the provisioning of your private network before enabling the toggle.
+
+Toggling the *App-Level Egress Rules* feature enables the *Default-Egress-Rule-Group*. The default egress rule is *Allow any protocol to any destination*. 
+
+image::ps-toggle-app-level-egress.png[Toggle app-level egress rules feature]
+
+To modify the default egress rule group:
+
+. Click *Edit*.
++
+The rule group configuration window opens.
+. In the *Egress Rules* section, use the drop-down list to select the desired protocol and destination.
+. Click *Add Rule* to add new rules to the rule group.
++ 
+You can delete rules by clicking the delete button image:trash-icon.png[Delete button, 2%,2%].
+. Click *Update Egress Rules* to save your changes.
++
+Your changes will affect all the applications in the current private space.
++
+[NOTE]
+You can neither rename nor delete the default rule group.
+
+image::ps-egress-edit-default.png[Edit the default app level egress rule group]
+
+To create a new custom egress rule group:
+
+. Click *Create Rule Group*.
++
+The rule group configuration window opens.
+. Add a group name.
++ 
+You can edit and rename custom rule groups. 
+. In the *Egress Rules* section, click *Add Rule* and use the dropdown list to select the desired protocol and destination.
+. Click *Add Rule* to add new rules to the new rule group.
```

Comment:
Is this correct? Does the user have to click *Add Rule* twice (once in the previous step and once here)?